src/h5web/visualizations/shared/Axis.tsx

```diff
@@ -9,6 +9,9 @@ const AXIS_PROPS = {
   tickStroke: 'grey',
   hideAxisLine: true,
   tickClassName: styles.tick,
+  labelClassName: styles.label,
+  labelProps: {}, // To avoid any styling props on the parent svg
```

Comment:
Otherwise the parent `svg` has `font-size=10` which makes a tiny tiny label due to `font-size: inherit` from `styles.label`.